src/includes/sourcemaps-wizard-instructions.mdx

```diff
@@ -0,0 +1,17 @@
+```bash
+npx @sentry/wizard@latest -i sourcemaps
+```
+
+The wizard will guide you through the following steps:
+
+- Connecting your local project to Sentry
```

Comment:
What exactly does this mean?

Reply:
This point should describe the login and project selection flow. I was thinking about how this makes sense from a user perspective and thought that the overall goal is to establish a connection/linking between the local project and the sentry project. However, I get how this can be a bit misleading. I'll reword it.